### DISCUSSION TITLE
Multiple weatherflow hubs

### DISCUSSION THREAD

**Discussion by Vertikar** [briis/hass-weatherflow2mqtt#64] [Ideas]
I've got a Tempest and an SWS, it seems that it doesn't differentiate between the UDP broadcasts from the devices, and lumps the temperature etc into the same MQTT topic, which then ends up with some values bouncing around between the two.
Is there any way currently to split separate hubs, or even just devices into their own topics?

**Comment by briis** [comment]
To clarify: Do you have 2 Hubs that broadcast? 

As far as I know, you can add more than 1 device to the same hub. That will give you double sensors in HA, but they are linked to the serial number in HA, so the data should not bounce.

**Reply by Vertikar** [reply]
Hi, 
Yep that's correct. One is running the older SWS, and another is running Tempest.
Apparently the older devices are not supported on the Tempest hubs.
